op-e2e/testdata/challenge_test_data.go

```diff
@@ -38,51 +38,53 @@ var (
-		Coinbase:    CoinbaseAddr,
-		Difficulty:  common.Big0,
-		Root:        common.HexToHash("0x1370c09d12e3aefefbe29bcecaa9a1adac759ee1b6657065f8e103f56b364037"),
-		TxHash:      common.HexToHash("0x2bd4ac406e80d8401dcd8a3770eac5630ff2883931d1acc6fdd86ece3a23c4d9"),
-		ReceiptHash: common.HexToHash("0xf75fc90d6167f310f60db885364953174fcd99a498b8629ea197074022a0eb67"),
-		Bloom:       types.BytesToBloom(hexutil.MustDecode("0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")),
-		Number:      big.NewInt(int64(TargetBlockNumber)),
-		GasLimit:    hexutil.MustDecodeUint64("0x1c9c380"),
-		GasUsed:     hexutil.MustDecodeUint64("0x10371"),
-		Time:        hexutil.MustDecodeUint64("0x64e4b763"),
-		Extra:       hexutil.MustDecode("0x"),
-		MixDigest:   common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000000"),
-		BaseFee:     big.NewInt(int64(hexutil.MustDecodeUint64("0x3a61b96"))),
+		ParentHash:      common.HexToHash("0x936a872193703be4201cffc531b00fd6e8c71a092c0e7ea51ab0f8ece00838a4"),
+		Coinbase:        CoinbaseAddr,
+		Difficulty:      common.Big0,
+		Root:            common.HexToHash("0x245d2fa3cbb92f5223e06b23a4c53fc1e1b9cd8edea6769e6749e72c8ff2d384"),
+		TxHash:          common.HexToHash("0xc9862b8f3157beea2d258b0a36b836794ea58302951468ab3d0b0885e63bccbf"),
+		ReceiptHash:     common.HexToHash("0x1ce54d356f9c39a93aac476d492b27c4c23374e203d9902becd4f4b6826b9b35"),
+		Bloom:           types.BytesToBloom(hexutil.MustDecode("0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")),
+		Number:          big.NewInt(int64(TargetBlockNumber)),
+		GasLimit:        hexutil.MustDecodeUint64("0x1c9c380"),
+		GasUsed:         hexutil.MustDecodeUint64("0xc0b1"),
+		Time:            hexutil.MustDecodeUint64("0x65f2c15d"),
+		Extra:           hexutil.MustDecode("0x"),
+		MixDigest:       common.HexToHash("0x0000000000000000000000000000000000000000000000000000000000000000"),
+		BaseFee:         big.NewInt(int64(hexutil.MustDecodeUint64("0x1"))),
+		WithdrawalsHash: &withdrawalHash,
```

Comment:
How about using `types.EmptyWithdrawalsHash`?
```suggestion
		WithdrawalsHash: &types.EmptyWithdrawalsHash,
```